src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final OrderResponseMapper orderResponseMapper;
+    private final ShoppingCartService shoppingCartService;
+
+    @Autowired
+    public OrderController(OrderService orderService, UserService userService,
+                           OrderResponseMapper orderResponseMapper,
+                           ShoppingCartService shoppingCartService) {
+        this.orderService = orderService;
+        this.userService = userService;
+        this.orderResponseMapper = orderResponseMapper;
+        this.shoppingCartService = shoppingCartService;
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> getOrdersByUser(@RequestParam Long id) {
+        return orderService.getOrdersHistory(userService.get(id))
+                .stream()
+                .map(orderResponseMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping("/complete")
+    public OrderResponseDto create(@RequestParam Long userId) {
```

Comment:
```suggestion
    public OrderResponseDto completeOrder(@RequestParam Long userId) {
```
Fix everywhere. Create is not informative name